src/main/scala/com/codacy/api/client/CodacyClient.scala

```diff
@@ -68,27 +69,39 @@ class CodacyClient(
    * Does an API post
    */
   def post[T](request: Request[T], value: String)(implicit reads: Reads[T]): RequestResponse[T] = {
-    val headers = tokens ++ Map("Content-Type" -> "application/json")
-
-    val body = Http(s"$remoteUrl/${request.endpoint}")
-      .params(request.queryParameters)
-      .headers(headers)
-      .postData(value)
-      .asString
-      .body
-
-    parseJsonAs[T](body)
+    val url = s"$remoteUrl/${request.endpoint}"
+    try {
```

Comment:
Codacy found an issue: [Consider using util.Try instead of a try-catch block](https://app.codacy.com/gh/codacy/codacy-api-scala/pullRequest?prid=6119275)